Containers/talk/Dockerfile

```
@@ -61,6 +61,9 @@ RUN mkdir -p /etc/nats; \
 # Give root a random password
 RUN echo "root:$(openssl rand -base64 12)" | chpasswd
 
+# Set default talk port https://github.com/nextcloud/all-in-one/issues/1011
+ENV TALK_PORT=3478
```

Can you please adjust this to this? (I think it is required to write it like this)

```suggestion
ENV TALK_PORT 3478
```

---

Oh, I see both is possible

---

all right! then fine by me!